_docs/admin-self-hosted.md

```diff
@@ -0,0 +1,9 @@
+---
+title: As an Admin (GitLab Self Hosted Repository)
+subtitle: This section will guide you in setting up your team for the GitLive plugin as an administrator.
```

Comment:
subtitle: This section will guide you in setting up your GitLive organization through your self-hosted repository hosting service ( Currently GitLve only support GitLab self-hosted)